pkg/splunk/enterprise/clustermanager.go

```diff
@@ -431,8 +432,8 @@ func getClusterManagerList(ctx context.Context, c splcommon.ControllerClient, cr
 	return numOfObjects, nil
 }
 
-// VerifyCMisMultisite checks if its a multisite
-func VerifyCMisMultisite(ctx context.Context, cr *enterpriseApi.ClusterManager, namespaceScopedSecret *corev1.Secret) ([]corev1.EnvVar, error) {
+// VerifyCMisMultisite checks if its a multisite used also in mock
+var VerifyCMisMultisiteCall = func(ctx context.Context, cr *enterpriseApi.ClusterManager, namespaceScopedSecret *corev1.Secret) ([]corev1.EnvVar, error) {
```

Comment:
Can we retain the name as `VerifyCMisMultisiteCall`? Same for other functions below. As we are changing the functionality for certain UT cases only?